whois-update/src/test/java/net/ripe/db/whois/update/handler/response/ResponseFactoryTest.java

```diff
@@ -621,4 +629,16 @@ private void assertVersion(final String response) {
                 "Handled mail update \\(TEST, 1970-01-01 00:00:00\\)\n" +
                 ".*"));
     }
+
+    private void assertVersionWithoutContact(final String response) {
```

Comment:
Is there any need for this method, if assertNotification already checks the content?